FeedStoreChallenge/CoreData/PersistentContainer.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+import CoreData
+
+final class PersistentContainer: NSPersistentContainer {
+
+    private struct CoreDataInitError: Error {}
+
+    static private let modelName = {
+        return "FeedStore"
+    }()
```

Comment:
`static let` is lazy by default. So you can simplify this declaration:

```suggestion
    private static let modelName = "FeedStore"
```

Reply:
👍